sdk/platform/src/persistence/mongodb/analytics.rs

```diff
@@ -63,7 +57,8 @@ impl Pipeline {
         let options = AggregateOptions::builder().build();
 
         // Map the stages over to Documents
-        let doc_pipeline = self.stages
+        let doc_pipeline = self
+            .stages
```

Comment:
Same comment here - Is this enforced with a linter rule? If not, can it be?

Reply:
See previous comment about linter and splitting large commits.